skymp5-server/cpp/server_guest_lib/ActionListener.cpp

```diff
@@ -456,8 +456,37 @@ void UseCraftRecipe(MpActor* me, const espm::COBJ* recipeUsed,
   spdlog::info("Using craft recipe with EDID {} from espm file with index {}",
                recipeUsed->GetEditorId(cache), espmIdx);
 
+  std::vector<Condition*> conditions;
```

Comment:
raw pointer detected 🤖

please use `std::unique_ptr`/`std::shared_ptr` instead

This is how we usually do it + I see incorrect `delete` logic which is much easier to replace than fix, re-review, etc